applications/app/controllers/LatestIndexController.scala

```diff
@@ -15,7 +15,11 @@ object LatestIndexController extends Controller with ExecutionContexts with impl
   def latest(path: String) = Action.async { implicit request =>
     loadLatest(path).map { _.map { index =>
       index.page match {
-        case tag: Tag if tag.isSeries || tag.isBlog => index.trails.headOption.map(latest => Found(latest.metadata.url)).getOrElse(NotFound)
+        case tag: Tag if tag.isSeries || tag.isBlog => index.trails.headOption.map(latest => {
+          if (request.isEmail) Found(latest.metadata.url + "/email")
```

Comment:
We just need to check whether Exact Target follows this before scraping, or whether we need to keep the url masked. The best way to find out will be release anyways!